packages/core/types/src/http/order/common.ts

```diff
@@ -98,16 +103,26 @@ interface BaseOrderShippingMethod {
   metadata: Record<string, unknown> | null
   tax_lines?: BaseOrderShippingMethodTaxLine[]
   adjustments?: BaseOrderShippingMethodAdjustment[]
+  original_total: BigNumberValue
```

Comment:
Note: I think the totals are optional and present if selected, maybe they should be marked optional and selectable ?

Reply:
@adrien2p I took it from the orderDTO, so I assumed it's correct. Since this is an order, maybe they are stored in the DB at this point, and always available? I'll check just to be sure

Reply:
Sure, @carlos-r-l-rodrigues will know better for that part

Reply:
Nop they are not stored, they are always calculated. Let me update it.